JoelStansbury/das
# added outlook module

## Checkout Code
```bash
git fetch
git checkout outlook
```
## Testing
* `pip install -e .` (this will get `pywin32` so it can open outlook)
* `python -m das.app` (to launch the app)
* goto http://127.0.0.1:5000/main
* click on the `click me to run the get_accounts function` button
* ensure all outlook accounts are printed below
